components/settings/settings.cpp

```diff
@@ -393,17 +349,18 @@ std::string Manager::getString(const std::string &setting, const std::string &ca
 
 float Manager::getFloat (const std::string& setting, const std::string& category)
 {
-    return parseFloat( getString(setting, category) );
+    return std::atof(getString(setting, category).c_str());
```

Comment:
Using C atof instead of C++11 stof (atoi instead of stoi) will cause undefined behavior if the string is not a valid numerical value. I think it's better to use the C++ function and catch its two exceptions using 0 as a fallback to simulate the previous behavior of parse<type> functions.
Even then, stof is bad because it's locale-dependent, so you should use boost::lexical_cast<float> instead and catch its single exception.

Reply:
Use stoi/stof for now and handle possible exceptions.